### PR TITLE
Win11 fixes

### PR DESCRIPTION
**Pull request by miczyg1**

### CODE REVIEW

File: configs/config.protectli_cml_vp4670 [hunk 1]
```diff
@@ -25,7 +26,6 @@ CONFIG_TPM_MEASURED_BOOT=y
 CONFIG_BOOTMEDIA_LOCK_CONTROLLER=y
 CONFIG_BOOTMEDIA_LOCK_WPRO_VBOOT_RO=y
 CONFIG_BOOTMEDIA_LOCK_IN_VERSTAGE=y
-CONFIG_BOOTMEDIA_SMM_BWP=y
```
Comment (mkopec) [inline configs/config.protectli_cml_vp4670:28]: We keep it enabled for VP4630 and VP4650, but not VP4670?

File: src/mainboard/protectli/vault_cml/mainboard.c [hunk 2]
```diff
@@ -50,6 +55,36 @@ static void mainboard_final(void *unused)
...
 	.final = mainboard_final,
 };
+
+void mainboard_silicon_init_params(FSPS_UPD *supd)
+{
+	supd->FspsTestConfig.C1StateAutoDemotion = 0;
+	supd->FspsTestConfig.C1StateUnDemotion = 0;
+	supd->FspsTestConfig.C3StateAutoDemotion = 0;
+	supd->FspsTestConfig.C3StateUnDemotion = 0;
+}
```
Comment (mkopec) [inline src/mainboard/protectli/vault_cml/mainboard.c:90]: missing EOL

File: configs/config.protectli_cml_vp4670 (hunk 1 above)
Comment (miczyg1) [reply configs/config.protectli_cml_vp4670:28]: Of course not... Disabled SMM BWP for all CML platforms.

File: src/mainboard/protectli/vault_cml/mainboard.c (hunk 2 above)
Comment (miczyg1) [reply src/mainboard/protectli/vault_cml/mainboard.c]: Fixed

**Review by mkopec — DISMISSED** [review]

**Pull request merged**